guides/micronaut-data-jdbc-repository/metadata.json

```
@@ -3,15 +3,14 @@
   "slug": "micronaut-data-jdbc-repository",
   "title": "Access a database with Micronaut Data JDBC",
```

```suggestion
  "title": "Access a Database with Micronaut Data JDBC",
```

I think these titles may show up in all caps, but I put this in title case on principle.